packages/salesforce/src/Adaptor.js

```diff
@@ -548,9 +584,9 @@ export function create(sObject, attrs) {
  * })
  * @function
  * @param {boolean} logical - a logical statement that will be evaluated.
- * @param {String} sObject - API name of the sObject.
- * @param {Object} attrs - Field attributes for the new object.
- * @returns {Operation}
+ * @param {string} sObject - API name of the sObject.
+ * @param {(object|object[])} attrs -  The data to process, either a single object or an array of objects.
```

Comment:
I don't think the brackets `()` are needed? Do they have meaning?

I don't know how we handle multiple types in other adaptors. I'm not at my desk today, can you check?

This isn't a formal declaration so we could say `object or array of objects`. I'd even be happy with `object or array`

Reply:
@josephjclark the `()` is used to group union types. See jsdoc here https://jsdoc.app/tags-type

Reply:
I have updated the description to say `object or array of objects` because it's an array we are expecting it to be an array of objects